content/user/troubleshooting/debug_logs.md

```diff
@@ -88,6 +88,7 @@ Some logs will follow special file naming conventions to make them more easily i
 
 - `myDEBUG-bounced-email-adm-` failure to send email from admin
 - `myDEBUG-bounced-email-` failure to send email from storefront
+- `zcInstallLog_` - log created by the `zc_install` process.
```

Comment:
Two notes:
- `zcInstallLog_*` files contain status details of inspection/install/upgrade operations, and are auto-pruned (removed) upon logging into the Admin console
- `zcDEBUG-*` files report fatal errors which occurred during the zc_install process (and are not auto-pruned)

Reply:
Thank you!  I thought I didn't have this detail quite right.